tests/test_conditional.py

```diff
@@ -0,0 +1,27 @@
+from codemod_tox.conditional import ToxConditional
+from codemod_tox.env import ToxEnv
+from codemod_tox.options import ToxOptions
+
+
+def test_conditional():
+    c = ToxConditional.parse("x\ny: z")
+    assert c.lines == (
+        (None, "x"),
+        (ToxEnv(("y",)), "z"),
+    )
+    assert str(c) == "x\ny: z"
+    assert c.evaluate("foo") == "x"
+    assert c.evaluate("yy") == "x"
+    assert c.evaluate("y") == "x\nz"
```

Comment:
I would have thought that `c.evaluate("y") == "z"` but that's not the case. I'm confused.

Reply:
The `x` line is unconditional, since there's no colon.  For example

```
deps = -rrequirements.txt
       -rtests/requirements.txt
       flask: flask>0
       fastapi: fastapi>0
```

for any env with a `flask` factor is the two `-r` lines plus the `flask>0` line.  If you had some theoretical `py38-flask-fastapi` env that has both factors, it would include all four lines.